src/ts/components/subtitleoverlay.ts

```diff
@@ -516,19 +528,80 @@ class SubtitleRegionContainer extends Container<ContainerConfig> {
     }, this.config);
   }
 
-  addLabel(labelToAdd: SubtitleLabel) {
+  private addCueBoxContainer(labelToAdd: SubtitleLabel) {
+    const id = labelToAdd.getConfig().id;
+    this.cueBoxContainers[id] = new SubtitleCueBoxContainer();
```

Comment:
Is there always just a 1-to-1 relationship as you always create a new container?